style.css

```diff
@@ -65,6 +64,191 @@ li {
   font-style: normal;
   padding-top: 10%;
   padding-bottom: 10%;
+  line-height: 52px;
+}
+
+.second-section {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  width: 327px;
+  height: 620px;
+}
+
+.third-section {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  width: 327px;
+  height: 640px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.fourth-section {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  width: 327px;
+  height: 640px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.fifth-section {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  width: 327px;
+  height: 640px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.sixth-section {
+  width: 375px;
+  height: 1000px;
+  background: #fff;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+}
+
+#my-btn {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  padding: 12px;
+  background: #fff;
+  border: 1px solid #396df2;
+  border-radius: 12px;
+}
```

Comment:
- [ OPTIONAL] The previous reviewer would have requested change if he saw this but Please make the  `See project`  `button` follow the button styling: Active, Hover, and Pressed (see design)

![Screenshot from 2022-12-03 19-12-27](https://user-images.githubusercontent.com/92303891/205457943-227f0990-7771-43ac-9c9c-dc7f6cdd5c46.png)